odoo/addons/base/models/res_partner.py

```diff
@@ -63,6 +63,23 @@ def _view_get_address(self, arch):
                 address_node.getparent().replace(address_node, sub_arch)
         return arch
 
+    @api.model
+    def _get_view_cache_key(self, view_id=None, view_type='form', **options):
```

Comment:
Not related to this code, but in ``_view_get_address`` can't we skip ``self.env.company.country_id.address_view_id.sudo()`` if ``self._context.get('no_address_format')`` is True ? Aka early return in that case, would be clearer ?

Looking at the code, not sure we gain anything from performance point of view, as it is used to skip children address view usage and therefore probably already in cache, just pointing it out.

Or we leave that code as it is in 16, which is fine also ^^ .